www/ToNative.js

```diff
@@ -45,17 +45,11 @@ function onInitializeFailed(error) {
         message = prefix + error + suffix;
     }
 
-    alert(message);
```

Comment:
But at least a console.error could be used? That way we could attach a browser to the device and see the apps console and the error message